src/main/java/com/aws/greengrass/mqttbridge/BridgeConfig.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.mqttbridge;
+
+import com.aws.greengrass.componentmanager.KernelConfigResolver;
+import com.aws.greengrass.config.Topics;
+import com.aws.greengrass.util.Coerce;
+import com.aws.greengrass.util.Utils;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Utilities for accessing MQTT Bridge configuration.
+ */
+public final class BridgeConfig {
+
+    @Deprecated
+    public static final String KEY_DEPRECATED_BROKER_URI = "brokerServerUri";
+    public static final String KEY_BROKER_URI = "brokerUri";
+    public static final String KEY_CLIENT_ID = "clientId";
+    public static final String KEY_MQTT_TOPIC_MAPPING = "mqttTopicMapping";
+
+    public static final String[] PATH_DEPRECATED_BROKER_URI =
+            {KernelConfigResolver.CONFIGURATION_CONFIG_KEY, KEY_DEPRECATED_BROKER_URI};
```

Comment:
Why do we have a deprecated field? I agree that brokerUri is better, but deprecating a field is nontrivial

Reply:
according to https://github.com/aws-greengrass/aws-greengrass-mqtt-bridge/pull/48, looks like `brokerServerUri` was once removed but added back for backward compatibility.   

the motivation behind marking it as `@deprecated` is to discourage usage

Reply:
Ended up renaming constants to `KEY_BROKER_SERVER_URI`/`PATH_BROKER_SERVER_URI` for clarity and left comment about backwards compatibility